src/lib/loaders/loaders_with_authentication/gravity.js

```diff
@@ -128,5 +128,10 @@ export default (accessToken, userID, opts) => {
       {},
       { method: "DELETE" }
     ),
+    usersLoader: gravityLoader(
+      "users",
+      {},
+      { headers: true }
```

Comment:
This is useful when you want to resolve with body and headers. You might want to resolve this way if you are paginating and need total count (accessible in the header, for example). Otherwise you may not need this.